### PR TITLE
improve experience when copying an app with static UCRs in mobile reports

### PR DESCRIPTION
**Pull request by nickpell**
https://dimagi-dev.atlassian.net/browse/HI-549

🐡

### CODE REVIEW

File: corehq/apps/app_manager/views/apps.py [hunk 1]
```diff
@@ -460,10 +460,10 @@ def _copy_app_helper(request, master_domain, master_app_id_or_source, copy_to_do
     try:
         app_copy = import_app_util(master_app_id_or_source, copy_to_domain, extra_properties)
     except ReportConfigurationNotFoundError:
-        messages.error(request, _("Copying the application failed because "
-                                  "your application contains a Report Module "
-                                  "that references a static UCR configuration."))
-        return HttpResponseRedirect(reverse_util('app_settings', params={}, args=[master_domain, app_id]))
+        messages.warning(request, _("Copying the application succeeded, but the application will have errors "
+                                    "because your application contains a Report Module that references "
+                                    "a static UCR configuration.")
```
Comment (jessica-muremure) [inline corehq/apps/app_manager/views/apps.py]: Can we say, "a **Mobile** Report Module that references a **UCR configured in this project space**"? Static UCR means something else, to me.

Comment (nickpell) [reply corehq/apps/app_manager/views/apps.py]: Done!

File: corehq/apps/app_manager/views/apps.py [hunk 2]
```diff
@@ -460,10 +460,9 @@ def _copy_app_helper(request, master_domain, master_app_id_or_source, copy_to_do
     try:
         app_copy = import_app_util(master_app_id_or_source, copy_to_domain, extra_properties)
     except ReportConfigurationNotFoundError:
```
Comment (esoergel) [inline corehq/apps/app_manager/views/apps.py]: Does whatever raises this exception not short-circuit the rest of the copying process?

Comment (proteusvacuum) [reply corehq/apps/app_manager/views/apps.py]: yeah, this is confusing...

Comment (nickpell) [reply corehq/apps/app_manager/views/apps.py]: The copying process gets as far as:
https://github.com/dimagi/commcare-hq/blob/ab43130d709daf8ae3129fb985f6181ad7c55dd9/corehq/apps/app_manager/models.py#L5858

The only thing that would be broken is multimedia.  Based on this previous ticket:
https://dimagi-dev.atlassian.net/browse/ICDS-240
it's only advanced Dimagi users that will ever run into this issue.  Probably fine if a warning about multimedia is added as well.

Comment (nickpell) [reply corehq/apps/app_manager/views/apps.py]: Updated: e85e2a1ccf8

Comment (esoergel) [reply corehq/apps/app_manager/views/apps.py]: This still feels like an odd place for this check - could this check be moved to the `import_app` function, perhaps?  That way it's caught where it's expected to be raised and it's clear what's being skipped.  I understand that'd make this message harder to display, maybe pass back a list of "warnings" that can be shown to the users or something.  IDK, that's not a perfect solution either.

Comment (nickpell) [reply corehq/apps/app_manager/views/apps.py]: Here's an alternative approach: 3a430de54a4

**Review by esoergel — APPROVED** [review]

**Pull request merged**